packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/launcher/loaders/DevLauncherAppLoader.kt

```diff
@@ -59,6 +59,9 @@ abstract class DevLauncherAppLoader(
           onReactContext(context)
           appHost.reactInstanceManager.removeReactInstanceEventListener(this)
           reactContextWasInitialized = true
+          synchronized(DevLauncherController.instance) {
+            DevLauncherController.instance.appIsLoading = false
+          }
```

Comment:
What do you think about creating an `onAppLoaded` method on DevLauncherController that does this and calling it here instead? It would be nice to keep the interdependencies between these classes as simple and clean as possible, and it's conceivable that callback could be used for other things too.